.github/workflows/release.yml

```diff
@@ -27,4 +29,48 @@ jobs:
           version: latest
           args: release --rm-dist
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
\ No newline at end of file
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  docker-release:
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      -
+        name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v3
+        with:
+          # list of Docker images to use as base name for tags
+          images: |
+            ${{ secrets.DOCKERHUB_USERNAME }}/headsacle
```

Comment:
Is this a generic variable that will resolve to juanfont? Or will this set it up to release under your name?

Reply:
Sorry just saw the issue, it will resolve to the headscale user on dockerhub from my understanding?

And then follow the repo on GitHub?

Reply:
```suggestion
            ${{ secrets.DOCKERHUB_USERNAME }}/headscale
```

Reply:
The DOCKERHUB_USERNAME can be anything we would like to. My suggestion would be headscale/headscale as mentioned in the issue.  So the DOCKERHUB_USERNAME would be in that case also headscale

Reply:
Cool, sounds reasonable, I was mostly seeking continuity/integrity by either using headscale or juanfont so people understand its the official image.

Does this need @juanfont  to add a secret before merging?

Reply:
Yeah @juanfont would have to add secrets to the repo first. (I also would hand over the dockerhub account to him) Maybe we can also think about moving the project into an organization. Probably we should discuss that.

Reply:
> Maybe we can also think about moving the project into an organization. Probably we should discuss that.

I suppose this can be created as a issue for discussion, I personally dont see any reason and I believe having it like this gives @juanfont the appropriate credit for implementing the project.

Reply:
Yeah that is a good idea to move it to a separate issue.